pandas/core/series.py

```diff
@@ -4775,7 +4778,7 @@ def _convert_dtypes(
 
     @doc(NDFrame.isna, klass=_shared_doc_kwargs["klass"])
     def isna(self) -> "Series":
-        return super().isna()
+        return generic.NDFrame.isna(self)
```

Comment:
wouldn't you directly cal the one in missing itself?

Reply:
actually doing the opposite for duplicated, so i think ok to in-line it here for consistency

Reply:
we're specifically inheriting the doc above

Reply:
ok pls consider to unfiy the calling covention with other methods , e.g. we either in-line things *here* or in a mixin or whatever to be consistent.